server.js

```diff
@@ -34,47 +37,61 @@ function search(origin, destination){
   return knex('listings').where(searchObject).innerJoin('users', 'listings.userID', '=', 'users.userID')
 }
 
+function singleListing(listingID){ // check if needed
```

Comment:
it is being used. 
let's talk about where we're extracting all our db methods to.